frontend/src/App.svelte

```diff
@@ -61,7 +62,11 @@
     rootResourceLoadPromise = new Promise((resolve) => {}),
     resourceNodeDataMap = {},
     resources = {};
-  let carouselSelection, currentResource, rootResource, modifierView;
+  let carouselSelection,
+    currentResource,
+    rootResource,
+    modifierView,
+    scriptView;
```

Comment:
`modifierView` ended up becoming a bit of a misnomer. I don't want the same fate for `scriptView` that will result in a refactor later. Probably worth renaming to something like `bottomLeftPane` or something. Someone will refactor the `modifierView` name into something clearer at a later date.

Moreover, it's confusing to have a variable named `scriptView` and a class named `ScriptView` – seems ripe for unexpected errors.